input/fsh/PractitionerSECommonsLite.fsh

```diff
@@ -20,3 +20,75 @@ Description: "This is a light weight Swedish Practitioner profile.
 * telecom.value 1.. MS
 
 Alias: $SEBaseHumanName = http://hl7.se/fhir/ig/base/StructureDefinition/SEBaseHumanName
+
+Instance: minimal-practitioner
+InstanceOf: PractitionerSECommonsLite
+Description: "A simple example of a Swedish Practitioner resource."
+Usage: #example
+* meta.profile = Canonical(PractitionerSECommonsLite)
+* identifier
+  * use = #official
+  * type = IDENTIFIER_TYPE#PRN
```

Comment:
According to the profile this should be

http://terminology.hl7.org/CodeSystem/v2-0203#PRN

Reply:
Which I know see is solved using aliases :)